include/libswiftnav/ephemeris.h

```diff
@@ -18,6 +18,9 @@
 #include <libswiftnav/time.h>
 #include <libswiftnav/common.h>
 
+#define SV_TYPE_GLONASS    0
+#define SV_TYPE_GLONASS_M  1
```

Comment:
We currently don't capture this for GPS (but there is a PR for it in the L2C stuff). Should we store it the same way?